docs/experimental-features.md

```diff
@@ -34,7 +34,7 @@ interface Vlan3
 
 ## Unified diff
 
-Starting in version 2.1.0, a featured called unified diff was introduced. It provides a similar output to difflib.unified_diff() but is aware of out of order lines and the parent child relationships present in the hier_config model of the configurations being diffed.  
+Unified diff was introduced in version 2.1.0. It provides a similar output to difflib.unified_diff() but is aware of out of order lines and the parent child relationships present in the hier_config model of the configurations being diffed.  
```

Comment:
May I suggest:

Unified diff was introduced in version 2.1.0. It provides a similar output to difflib.unified_diff(), but is also aware of incorrectly ordered lines and the parent-child relationships present in the hier_config model of the configurations being diffed.